chart/openfaas/templates/psp.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if .Values.psp }}
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+    name: {{ .Release.Name }}-psp
+    labels:
+        app: {{ template "openfaas.name" . }}
+        chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+        heritage: {{ .Release.Service }}
+        release: {{ .Release.Name }}
+    annotations:
+        seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+spec:
+    privileged: false
+    hostIPC: false
+    hostNetwork: false
+    hostPID: false
+    readOnlyRootFilesystem: false
+    allowPrivilegeEscalation: true
```

Comment:
I'm quite concerned about this, why is it needed?

Surely Istio doesn't need to run envoy as privileged?

Reply:
Ah I've only disabled privileged, I'll fix it.